modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -28,11 +44,59 @@ export interface UploadResult {
 }
 
 interface EditorUpload {
+  /**
+   * Cache of blob elements created in an editor instance.
+   *
+   * @property blobCache
+   * @type Object
+   */
   blobCache: BlobCache;
+
+  /**
+   * Adds a custom filter to be executed before any images are uploaded to the server.
+   * Images must return true on every added filter to be uploaded, else they are filtered out.
+   *
+   * @method addFilter
+   * @param {Function} filter Function which filters each image upload.
+   * @example
+   * // Filter which images are uploaded.
+   * tinymce.activeEditor.EditorUpload.addFilter((image) => {
+   *   const maxSize = 1920 * 1080;
+   *   const imageSize = image.width * image.height;
+   *   return imageSize < maxSize;
+   * });
+   */
   addFilter: (filter: (img: HTMLImageElement) => boolean) => void;
+
+  /**
+   * Uploads all data uri/blob uri images in the editor contents to server.
+   *
+   * @method uploadImages
+   * @return {Promise} Promise instance with images and status for each image.
+   */
   uploadImages: () => Promise<UploadResult[]>;
+
+  /**
+   * Uploads all data uri/blob uri images in the editor contents to server only when automatic uploads are enabled.
```

Comment:
```suggestion
   * Upload all data uri/blob uri images in the editor contents to server only when automatic uploads are enabled.
```
Sticking to simple present transitive both for consistency’s sake and to keep the declarative approach clearer.